specs/MultiProfile.md

```diff
@@ -296,6 +306,8 @@ namespace Microsoft.Web.WebView2.Core
         Boolean IsInPrivateModeEnabled { get; };
 
         String ProfilePath { get; };
+
+        CoreWebView2CookieManager cookieManager { get; };
```

Comment:
This should be capital **C**ookieManager. If this was generated for you by build-apiwriter.ps1, then you may need to add CookieManager to the list under `# capitalize the first letter for properties` in buid-apiwriter.ps1.

```suggestion
        CoreWebView2CookieManager CookieManager { get; };
```